front/admin/relocate_users.ts

```diff
@@ -0,0 +1,107 @@
+import { removeNulls } from "@dust-tt/types";
+import type { ApiResponse } from "auth0";
+
+import { getAuth0ManagemementClient } from "@app/lib/api/auth0";
+import { SUPPORTED_REGIONS } from "@app/lib/api/regions/config";
+import { Authenticator } from "@app/lib/auth";
+import { MembershipResource } from "@app/lib/resources/membership_resource";
+import { UserResource } from "@app/lib/resources/user_resource";
+import { makeScript } from "@app/scripts/helpers";
+
+makeScript(
+  {
+    region: {
```

Comment:
`nit` To align with the relocate_workspace script.
```suggestion
    destinationRegion: {
```